### PR TITLE
PP-9361: Run pr e2e tests automatically 

### PR DESCRIPTION
**Pull request by jfharden**
Run all PR e2e tests automatically and without waiting for unit tests to pass first

### CODE REVIEW

File: ci/pipelines/pr.yml [hunk 1]
```diff
@@ -605,8 +605,7 @@ jobs:
     plan:
       - <<: *get-pull-request
         resource: card-connector-pull-request
-        trigger: false
-        passed: [card-connector-unit-test]
+        trigger: true
```
Comment (katstevens) [inline ci/pipelines/pr.yml:608]: We should probably be consistent and either have `trigger: true` explicitly written out for all the tasks (not just here), or remove this one?

Comment (jfharden) [reply ci/pipelines/pr.yml]: Removed 👍

**Review by katstevens — APPROVED** [review]
Woohoo let's gooooo 💨

**Comment by jfharden** [comment]
They pass on a new pr on master so I'm happy

**Pull request merged**